cellcounter/cc_kapi/README.md

```diff
@@ -12,9 +12,42 @@ cc_kapi is listed in the INSTALLED_APPS in settings.py:
 In urls.py we have a declaration to point the keyboard api related requests to the cc_kapi urls.py
 
     urlpatterns = patterns('',
-                           url(r'^api/keyboard/', include('cellcounter.cc_kapi.urls')),
+                           url(r'^api/keyboards', include('cellcounter.cc_kapi.urls')),
                            )
 
 Finally, in deployment, we'll need to run ```collectstatic``` in order to pull the Javascript into the main static directory.
 
-    python manage.py collectstatic
\ No newline at end of file
+    python manage.py collectstatic
+
+
+# Usage
+
+Access through api/keyboards/
+
+api/keyboards/
+    GET: lists all available keyboards, including builtin and user keyboards for both desktop and mobile
+
+api/keyboards/desktop/
+    GET: lists all available desktop keyboards, including builtin and user keyboards
+    POST: adds a new desktop keyboard
+
+api/keyboards/desktop/<id>/
+    GET: gets the desktop keyboard with identifier <id> (<id>: db_key | 'builtin' | 'default')
+    PUT: updates the specified desktop keyboard
+    DELETE: deletes the specified desktop keyboard
+
+api/keyboards/desktop/<id>/set_default
+    POST: sets the specified desktop keyboard as the default desktop keyboard
```

Comment:
The `api/keyboards/desktop/default` URL is only provided as a convenience for the front end. Whilst I understand this breaks the single point of access principle it does simplify use of the API somewhat. However, the resource has its "proper" URL at `api/keyboards/desktop/<id>` If you are really keen not to break single point of access then I would remove the `api/keyboards/desktop/default` URL rather than change the `set_default` method.

Therefore the second suggestion makes more sense to me, and I agree it should be a `PUT` rather than a `POST`. Changed in https://github.com/cellcounter/cellcounter/pull/446/commits/6baec572dc17acb7895c5459b2dc334fe6154684